### ISSUE TITLE
Wrapping with web mercator (and similar) CRSes

### ISSUE THREAD

**Issue by guygriffiths** [Reading-eScience-Centre/edal-java#98]
We can have coordinate reference systems which are not defined in terms of lat-long values, but which still need to wrap at the date line.
These are not used widely in ncWMS at the moment, but I spotted the issue when plotting wrapped trajectory data in web mercator CRS.
Probably needs to be implemented in the appropriate axis implementations, which currently assume that wrapping is synonymous with longitude axes.

**Comment by jonblower** [comment]
Yes, this is important but also a tricky one. Strictly speaking, in a projected CRS (like Web Mercator) there is no notion of wrapping; it only really appears in a spherical/ellipsoidal CRS. Any apparent wrapping is usually done in the client. But there is definitely a practical issue here - perhaps we could enumerate some possible solutions (e.g. should the logic exist on the client side or the server side?)
